lib/freno/client.rb

```diff
@@ -47,8 +47,8 @@ def initialize(faraday)
     #
     # Returns Result
     #
-    def check(app: default_app, store_type: default_store_type, store_name: default_store_name, options: self.options)
-      perform :check, app: app, store_type: store_type, store_name: store_name, options: options
+    def check(app: default_app, store_type: default_store_type, store_name: default_store_name, options: {})
```

Comment:
I feel like this changes the API in ways that consumers may not expect if they have changed `options` on the client?

Previously passing any options would override the client options, now it will extend them?

Reply:
Maybe that okay, and we just document the `client.options` somewhere, and specify that it'll get merged in?

Reply:
Done! I'll release a new minor version when it's time. The `freno-client` gem is still pre-1.0 so this will signal that breaking changes are still possible on minor version bumps.